src/components/Layout/OverviewList.tsx

```diff
@@ -10,6 +12,7 @@ import { HeaderMenu } from "decentraland-ui/dist/components/HeaderMenu/HeaderMen
 import Icon from "semantic-ui-react/dist/commonjs/elements/Icon"
 
 import { AggregatePlaceAttributes } from "../../entities/Place/types"
+import { useCardsByWidth } from "../../hooks/useCardsPerWindowSize"
```

Comment:
```suggestion
import useCardsByWidth from "../../hooks/useCardsByWidth"
```

following the convention: https://github.com/decentraland/decentraland-gatsby/blob/master/docs/guide/typescript.md#exports-named-vs-default-export